src/pre-commit.check_working_on_master.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+
+BRANCH_MASTER="master";
+
+if [[ $(git rev-parse --abbrev-ref HEAD) == $BRANCH_MASTER ]];
+then
+    echo "!!!WARNING!!! Now you working on the master branch."
+    exec < /dev/tty
+    while :
+    do
+        read -p "Are you really want to continue?  [y/n] " answer
+        case "$answer" in
+            y|yes)
+                echo -e  "Commit successful."
+                exit 0
+                ;;
+            n|no)
+                echo -e "Commit has failed."
```

Comment:
1. `Commit has failed."` -> `Commit was canceled."`
2. Please remove `-e` modifier, it's not needed here.

Reply:
1. changed
2. removed